node/resolvers/Queries/Organizations.ts

```diff
@@ -241,6 +241,111 @@ const Organizations = {
     }
   },
 
+  getOrganizationsByEmailPaginated: async (
+    _: void,
+    {
+      email,
+      page = 1,
+      pageSize = 25,
+    }: {
+      email: string
+      page: number
+      pageSize: number
+    },
+    {
+      clients: { storefrontPermissions, session },
+      vtex: { logger, sessionToken, adminUserAuthToken },
+    }: any
+  ) => {
+    const organizationFilters: string[] = []
+    let fromSession = false
+    const {
+      data: { checkUserPermission },
+    }: any = await storefrontPermissions
+      .checkUserPermission('vtex.b2b-organizations@1.x')
+      .catch((error: any) => {
+        logger.error({
+          error,
+          message: 'checkUserPermission-error',
+        })
+
+        return {
+          data: {
+            checkUserPermission: null,
+          },
+        }
+      })
+
+    if (
+      (!adminUserAuthToken &&
+        !checkUserPermission?.permissions.includes('add-sales-users-all')) ||
```

Comment:
why are we checking if the user has `add-sales-users-all` permission if this is just a `getOrganizationsByEmail` query?

Reply:
Hi @enzomerca 
Is the same approach of the getOrganizationsByEmail query. If I'm not mistaken this is for the case of a query run through the webstore where there's no admin auth cookie.